README.md

```diff
@@ -62,9 +62,12 @@ const [useCounter, sharedCounter$] = connectObservable(
   )
 )
 ```
-Returns a hook that provides the latest update of the accepted observable, and an enhanced 
-version of the original observable (the one that the hook will use), which is shared 
-and "replayable".
+Returns a hook that provides the latest update of the accepted observable, and the
```

Comment:
I actually prefered it before. We should assume users of this library know RX. Trying to describe this function in terms of subscribers and subscriptions is hard to read and a bit confusing IMO.

Reply:
TBH I was having a very hard time when I wrote that to find a precise and concise way to describe how the returned observable is being enhanced... At some point I just got lazy and I wrote that knowing that it was a lazy way to describe it. It's lazy because there are sooo many different ways to "share" and/or "replay" with RxJS... What I had in my head was: "well it's ok, because later on in the section for `distinctShareReplay` I will extend this explanation".

Anyways, I like @voliva explanation a lot better because while being pretty concise, he actually explains really well how exactly it is that the enhanced observable is sharing/replaying.

Can that description be confusing the first time that you read it? Quite possible. Is it accurate? Yep, it is. So, I much rather for the reader to read it a few times until it sinks than to provide a handwavy explanation, which is what I think that we got before.

For me the ideal docs are the ones that start providing good examples so that anyone can grok things, but that also provide the necessary detail so that experts can understand what's actually going on.

All that being said, I still think that there is lots of room for improvement. So, let's keep bringing good ideas and suggestions on how to improve the docs.